redaxo/src/addons/structure/update.php

```diff
@@ -21,6 +21,25 @@
     });
 }
 
+if (rex_addon::get('users')->isInstalled() && $addon->getPlugin('content')->isInstalled() && rex_version::compare($addon->getVersion(), '2.11.0-beta1', '<')) {
```

Comment:
```suggestion
if (rex_addon::get('users')->isInstalled() && $addon->getPlugin('content')->isInstalled() && rex_string::versionCompare($addon->getVersion(), '2.11.0-beta1', '<')) {
```

Das Update wird unter Umständen mit einer Core-Version durchgeführt, die `rex_version` noch nicht enthält.